src/apps/Registration.js

```diff
@@ -56,7 +57,15 @@ const RegistrationComponent = (props) => {
                         .auth()
                         .createUserWithEmailAndPassword(email, pass)
                         .then((user) => {
-                            alert("This is where Yashika works her magick");
+                            writeData();
+                            const docRef = firebase
+                                .firestore()
+                                .collection("Users")
+                                .doc();
+                            docRef.set({
+                                Name: document.getElementById("name").value,
```

Comment:
Also don't use `document.getElementById()` since we're using React, and that will cause issues if the document is modified outside of React. Instead, just use the `name` variable from line 9.
```suggestion
                                name: name,
```